### DISCUSSION TITLE
Inline math in markdown does not render when enclosed in parentheses

### DISCUSSION THREAD

**Discussion by cdwilson** [community/community#28115] [Other Feature Feedback, Questions, & Ideas]
```
1. This works $\sqrt{3x-1}+(1+x)^2$
2. This works ( $\sqrt{3x-1}+(1+x)^2$ )
3. This doesn't work ($\sqrt{3x-1}+(1+x)^2$)
```

Here's how it renders:

1. This works $\sqrt{3x-1}+(1+x)^2$
2. This works ( $\sqrt{3x-1}+(1+x)^2$ )
3. This doesn't work ($\sqrt{3x-1}+(1+x)^2$)

(a screenshot for posterity in case this issue gets fixed and the render above ^ starts looking correct)

<img width="386" alt="image" src="https://user-images.githubusercontent.com/14631/182498257-faca1394-84ea-45ee-b646-f6fa779fe8c8.png">

**Comment by nschloe** [comment]
It's not only parentheses. If at least one of the two dollar signs is touched by some other character, it won't render as math.

_Edit:_ I've added this to the math bug repo. https://github.com/nschloe/github-math-bugs#sumproduct-signs-in-inline-mode-or-fractions

**Comment by nschloe** [comment]
Parentheses work now. Other things (like quotation marks, "$\pi$") still don't.

**Comment by nschloe** [comment]
[One can now use inline math with backticks](https://github.blog/changelog/2023-05-08-new-delimiter-syntax-for-inline-mathematical-expressions/) to solve the problem, ($`\sqrt{3x-1}+(1+x)^2`$).
